values.yaml

```
@@ -1022,6 +1022,21 @@ ui:
     # @type: string
     annotations: null
 
+  # Configurations for displaying metrics in the UI.
+  metrics:
+    # Enable displaying metrics in the UI. The default value of "-"
+    # will inherit from global.metrics.enabled value. This value is only
+    # used if ui.enabled is set to true.
```

```suggestion
    # will inherit from global.metrics.enabled value.
```

Since this is inside the `ui:` stanza already I think this is implied for all of these.
If we feel its necessary to mention, maybe at the `ui.metrics` level?